minimisers/DREAM/functions/initializeDREAM.m

```diff
@@ -46,18 +42,15 @@
     [x] = boundaryHandling(x,paramInfo);
 end
 
-% Now evaluate the model ( = pdf ) and return fx
-fx = evaluateModel(x,DREAMPar,Meas_info,ratInputs);
+% Now evaluate the model ( = pdf ) and return log-likelihood
+log_L_x = calcLogLikelihood(x,DREAMPar,ratInputs);

```

Comment:
Would it be worth using `Log_L_x` as the output variable for `calcLogLikelihood` too?